### ISSUE TITLE
Ccannot interpolate variables containing the string "$0" in IE <= 11

### ISSUE THREAD

**Issue by kortaggio** [fnando/i18n-js#405]
# Steps to reproduce

```
# en.yml
en:
  product_price: "This widget costs %{widget_price}"
```

```
// main.js
I18n.t("product_price", {widget_price: "$0"})
```

**Expected Behaviour**

`This widget costs $0`

**Actual Behaviour**

`This widget costs %{widget_price}`

**Comment by PikachuEXE** [comment]
IE specific issue?
So can't be auto tested?

**Comment by PikachuEXE** [comment]
I can see some test cases at
https://github.com/fnando/i18n-js/blob/master/spec/js/translate.spec.js#L180
Can you try to see if you can reproduce the issue by changing the spec a bit?

**Comment by kortaggio** [comment]
@PikachuEXE Yes, this is an IE specific issue. I can reproduce on IE 11 but not on Chrome or Firefox. I'll take a look at the specs.

**Comment by fnando** [comment]
Just ran master tests and seems everything is working as expected. I even tried using `$0` as the interpolated value.

![](https://dl.dropboxusercontent.com/s/65rtm81e7lhtusq/2016-05-17%20at%209.34%20AM.png)

**Comment by kortaggio** [comment]
Looks like this is a reopening of https://github.com/fnando/i18n-js/issues/168
